api/src/orgs/schema.js

```diff
@@ -1,4 +1,18 @@
 export default `
+  # An organization's own profile.
+  type Profile {
+    # The human-readable name of the organisation.
+    name: String
+    # URL to the organisation's icon.
+    icon: String
+    # Array of URL to the different links for the organisation (ex: Discord, Discourse)
+    links: [String]!
+    # Array of addresses from the people that have edited the profile at least once
```

Comment:
```suggestion
    # Array of addresses that have edited the profile at least once
```

I would omit the people part since it might not be a person ( 🤖 ) and it is a bit verbose I think